### PR TITLE
Week 3

### PR DESCRIPTION
**Pull request by ShaikMoAbdullah**
- [x] Completed all the week 3 assignments

### CODE REVIEW

File: Week-3/assignment-4.5/index.html
Comment (NishantNaithani) [inline Week-3/assignment-4.5/index.html:1]: Here,
1. please use defined tags for the `text` type pf content.
2. class names should be meaningful.

File: Week-3/assignment-5.2/index.html
Comment (NishantNaithani) [inline Week-3/assignment-5.2/index.html:1]: Here,
1. under `body` tag we have added many tags for different purposes, but can be merged.

**Review by NishantNaithani — CHANGES_REQUESTED** [review]

**Comment by NishantNaithani** [comment]
Added comments

File: Week-3/assignment-5.2/index.html
Comment (ShaikMoAbdullah) [reply Week-3/assignment-5.2/index.html:1]: These tags were given in the template in LMS and we had to only make the required changes to achieve the desired UI result

File: Week-3/assignment-4.5/index.html
Comment (ShaikMoAbdullah) [reply Week-3/assignment-4.5/index.html:1]: Noted and made all the required changes.

**Review by NishantNaithani — APPROVED** [review]

**Pull request merged**